scripts/trelloOnboardingBuilder.js

```diff
@@ -83,6 +85,46 @@ var app = new Vue({
                 }
                 return timelineCardPosition;
             }
+            async function reorganizeCards(timelineCardPosition, trelloApiKey, trelloOAuth1) {
+                let reorganizedCardsCount = 0;
+                for (const card of allCardsInLearningPath) {
+                    if (! canMoveCardInSpecificTimeline(card.name)) { continue; }
+
+                    const timelineInformation = getTimelineInformation(card.name);
+                    let timelineInformationIndex = timelineInformationMapping[timelineInformation];
+                    if (! Object.keys(timelineCardPosition).includes(`${timelineInformationIndex}`)) { continue; }
+
+                    const expectedCardPositionInList = timelineCardPosition[timelineInformationIndex] + 1;
+                    incrementPositionsForTimelineInformationAfter(timelineInformationIndex, timelineCardPosition);
+                    await updateCard(card.id, { pos: expectedCardPositionInList }, { trelloApiKey, trelloOAuth1 });
+                    reorganizedCardsCount++;
+                }
+                return reorganizedCardsCount;
+            }
+            function canMoveCardInSpecificTimeline(name) {
+                return hasDayInformation() || hasWeekInformation() || hasMonthInformation();
+
+                function hasDayInformation() {
+                    return /.+\[DAY \d]/.test(name);
+                }
+                function hasWeekInformation() {
+                    return /.+\[WEEK \d]/.test(name);
+                }
+                function hasMonthInformation() {
+                    return /.+\[MONTH \d]/.test(name);
+                }
```

Comment:
@AdriAt360 or just `/.+\[DAY|WEEK|MONTH \d]/.test(name);` like below

Reply:
@pp0rtal I felt clearer to explain the content of these regex once, but I agree, it would also work
is it ok for you if I keep this version for documentation purpose?

Reply:
@AdriAt360 sure!